app/lib/pre_assembly/from_staging_location/structural_builder.rb

```diff
@@ -8,20 +8,23 @@ class StructuralBuilder
       # @param [Cocina::Models::DRO] cocina_dro
       # @param [String] reading_order
       # @param [Boolean] all_files_public
-      def self.build(filesets:, cocina_dro:, all_files_public:, reading_order: nil)
+      # @param [Boolean] manually_corrected_ocr set by user when creating the job
+      def self.build(filesets:, cocina_dro:, all_files_public:, reading_order: nil, manually_corrected_ocr: false)
         new(filesets:,
             cocina_dro:,
             reading_order:,
-            all_files_public:).build
+            all_files_public:,
+            manually_corrected_ocr:).build
       end
 
-      def initialize(filesets:, cocina_dro:, all_files_public:, reading_order: nil)
+      def initialize(filesets:, cocina_dro:, all_files_public:, reading_order:, manually_corrected_ocr:)
         @filesets = filesets
```

Comment:
note: nobody uses initialize except for `.build` above, and it already sets all the available defaults, no need to set here again (e.g. for `reading_order`)